docs-src/references/cluster-metrics.md

```diff
@@ -0,0 +1,177 @@
+---
+id: cluster-metrics
+title: Cluster metrics reference
+description: The Temporal Cluster reports a range of metrics to help operators get visibility into the Cluster’s performance and set up alerts.
+sidebar_label: Cluser metrics
+tags:
+  - reference
+---
+
+The Temporal Cluster reports a range of metrics to help operators get visibility into the Cluster’s performance and set up alerts.
+All metrics emitted by the Cluster are listed in [defs.go](https://github.com/temporalio/temporal/blob/master/common/metrics/defs.go).
+
+For details on setting up metrics in your Cluster configuraiton, see [Temporal Cluster](/references/configuration/##global).
+
+The [dashboards repository](https://github.com/temporalio/dashboards) contains community-driven Grafana dashboards that can be used for monitoring Temporal Server and SDK metrics.
+You can use these as a reference to build your own dashboards.
+For any metrics that are missing in the dashboards, use [defs.go](https://github.com/temporalio/temporal/blob/master/common/metrics/defs.go) as a reference.
+
+Note that apart from these metrics emitted by the Cluster, you should also monitor infrastructure-specific metrics like CPU, memory, and network for all hosts that are running Temporal Cluster services.
+
+## Common metrics
+
+Temporal emits metrics for each gRPC service request.
+These metrics are emitted with `type`, `operation`, and `namespace` tags, which provide visibility into service usage and show the request rates across services, namespaces, and even operations.
+
+- Use the `operation` tag with each metric in your query to get request rates, error rates or latencies per operation.
+- Use `service_name` tag with the [service role tag values](https://github.com/temporalio/temporal/blob/bba148cf1e1642fd39fa0174423b183d5fc62d95/common/metrics/defs.go#L108) to get details for the specific service.
+
+All common tags that you can add in your query are listed [here](https://github.com/temporalio/temporal/blob/86966c5d2f78ee74c10b6808857f1a2f64d3c134/common/metrics/defs.go#L90).
+
+For example, to see service requests by operation on Frontend, use:
+
+`sum by (operation) (rate(service_requests{service_name="frontend"}[2m]))`
+
+Note: All metrics queries in this article are [Prometheus queries](https://prometheus.io/docs/prometheus/latest/querying/basics/).
+
+The following list describes some metrics you can get started with.
+
+### [`service_requests`](https://github.com/temporalio/temporal/blob/bba148cf1e1642fd39fa0174423b183d5fc62d95/common/metrics/defs.go#L2497)
```

Comment:
```suggestion
### service_requests
```

I think we should not be linking headers. If it is not addressed already in the style guide, I would like to propose we add that.

Also at this point we want to generally move away from linking to specific lines in code if we can avoid it.

I think if we want to highlight where these metrics are defined in source, then maybe add a bold link at the top of the file?